### DISCUSSION TITLE
All user profiles show current user's friends

### DISCUSSION THREAD

**Discussion by Jamiras** [RetroAchievements/RAWeb#715] [Ideas]
As a user, when I view someone's profile, I expect to see their friends, not mine.

The "Friends Leaderboard" always shows the current user's friends. It should either show the friends of the user being viewed, or not be present when viewing a profile other than the current user's.

**Comment by kdecks** [comment]
I don't want my friends list to be public. This should be optional private, private by default.

**Comment by Ben-Ghazi** [comment]
Same, what reason is there for needing to see other people's friend list?

**Comment by Jamiras** [comment]
I'm just indicating that it's misleading as-is. It should either be corrected or removed.

**Comment by Jamiras** [comment]
This was indirectly handled in #441. The friends list was replaced with a friends leaderboard, and that is only visible when viewing the logged-in user's profile.
